### DISCUSSION TITLE
[A-L] (2023/24) Foglio 14 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#282] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1b37dc7d-37e9-42d1-bfda-09b6285a700c)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/22b148b5-da1f-42c0-b5ed-c7094b824352)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/c1a3e76b-1278-45de-97a1-dbc0855cdb38)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/77fdb30b-bf52-46b1-9896-d6055ae3243b)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/12b8ef08-35b1-4315-ac3d-5fd020920643)

**Comment by daromacarlo** [comment]
per dimostrare che Ʃₒ è sottospazio di R⁶ devo fare questo?
![Immagine 2023-12-07 111627](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/557cb101-4703-4901-b76d-b2b24a9ab462)
![Immagine 2023-12-07 111642](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/86acc64e-3c4a-48b6-afc7-2e03f45bd267)
![Immagine 2023-12-07 111708](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0ba0ce8f-5aff-4e34-96bf-c33a93559a9d)

**Reply by daromacarlo** [reply]
punto 3)
![Immagine 2024-01-07 221133](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/d13b4051-2c2e-4a4c-bc31-ebba15cd1ff0)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5f0b0d71-c07a-4a9f-8cfd-d22584ed7ea6)
